src/current/_includes/v23.1/essential-metrics.md

```diff
@@ -0,0 +1,178 @@
+| changefeed.failures               | changefeed.failures                                        | Total number of changefeed jobs which have failed            | This metric tracks the permanent changefeed job failures that the jobs system will not try to restart. Any increase in this counter should be investigated. An alert on this metric is recommended. |
+| changefeed.error_retries          | changefeed.error.retries                                   | Total retryable errors encountered by all changefeeds        | This metric tracks transient changefeed errors.  Alert on "too many" errors, such as 50 retries in 15 minutes. For example, during a rolling upgrade this counter will increase because the changefeed jobs will restart following node restarts. There is an exponential backoff, up to 10 minutes. But if there is no rolling upgrade in process or other cluster maintenance, and the error rate is high, investigate the changefeed job.
+| changefeed.emitted_messages       | changefeed.emitted.messages                                | Messages emitted by all feeds                                | This metric provides a useful context when assessing the state of changefeeds. This metric characterizes the rate of changes being streamed from the CockroachDB cluster. |
+| changefeed.emitted_bytes          | {% if include.deployment == 'self-hosted' %}changefeed.emitted_bytes.count |{% elsif include.deployment == 'dedicated' %} |{% endif %} Bytes emitted by all feeds                                   | This metric provides a useful context when assessing the state of changefeeds. This metric characterizes the throughput bytes being streamed from the CockroachDB cluster. |
+| changefeed.commit_latency         | changefeed.commit.latency                                  | The difference between the event MVCC timestamp and the time it was acknowledged by the downstream sink. If the sink batches events, then the difference between the oldest event in the batch and acknowledgement is recorded. Latency during  backfill is excluded.| This metric provides a useful context when assessing the state of changefeeds. This metric characterizes the end-to-end lag between a committed change and that change applied at the destination. |
+| jobs.changefeed.protected_age_sec | {% if include.deployment == 'self-hosted' %}jobs.changefeed.protected_age_sec |{% elsif include.deployment == 'dedicated' %} |{% endif %} The age of the oldest PTS record protected by changefeed jobs | [Changefeeds use protected timestamps to protect the data from being garbage collected](monitor-and-debug-changefeeds.html#protected-timestamp-and-garbage-collection-monitoring). Ensure the protected timestamp age does not significantly exceed the [GC TTL zone configuration](configure-replication-zones.html#replication-zone-variables). Alert on this metric if the protected timestamp age is greater than 3 times the GC TTL. |
+
+## Row-Level TTL
```

Comment:
The TTL-specific metrics from here should be added https://github.com/cockroachdb/cockroach/blob/fa4f9ba2752d81d133c319c0b9a8b4981a01d3d1/pkg/sql/ttl/ttljob/ttljob_metrics.go#L101

Reply:
Updated